src/redux/actions/index.js

```diff
@@ -34,3 +45,81 @@ export function setLanguageName(language) {
     payload: language,
   };
 }
+
+// Seen phrases with async storage
+export function setSeenPhrases(seenPhrases) {
+  return {
+    type: SET_SEEN_PHRASES,
+    payload: seenPhrases,
+  };
+}
+
+// Learnt phrases with async storage
+export function setLearntPhrases(learntPhrases) {
+  return {
+    type: SET_LEARNT_PHRASES,
+    payload: learntPhrases,
+  };
+}
+
+// Add learnt phrases
+export function addLearntPhrase(phrase) {
+  return async dispatch => {
+    const storedLearntPhrases = await getData(LEARNT_PHRASES_KEY);
+    let dataToStore = null;
```

Comment:
This is more elegant with a ternary:
```suggestion
    let dataToStore = storedLearntPhrase ? [...storedLearntPhrases, phrase] : [phrase]
```